docs/blog/write-the-docs-2018-stats.rst

```diff
@@ -0,0 +1,75 @@
+.. post:: Jan 7, 2018
+   :tags: stats, year in review
+   :author: Eric Holscher
+
+Write the Docs 2018 Stats
+=========================
+
+2018 was another wonderful year for Write the Docs, it marks our 6th year of existence.
+
+This post contains stats that mark our progress,
+so that we can look back and see how we've changed as a community.
+
+This year I've removed a lot of the commentary except where it's useful for interpreting the numbers.
+This makes it a simple mark of progress and lets us compare numbers across years in a consistent manner.
+
+.. note:: 
+   
+   You can also read our posts from 2016_ and 2017_.
+
+.. _2017: http://www.writethedocs.org/blog/write-the-docs-2017-stats/
+.. _2016: http://www.writethedocs.org/blog/write-the-docs-2016-year-in-review/
+
+Conferences
+===========
+
+* 450 attendees in Portland (from 400 in 2017)
+* 275 attendees in Prague (from 250 in 2017)
+* 100 attendees in Australia (from 50 in 2017)
+* 50 attendees in Cincinnati (New in 2018)
+
+Australia went from a one-day to a two-day conference,
+growing from 50 to 100.
+Cincinnati was new this year,
+in collaboration with OpenHelp,
+and got 50 attendees.
+
+Slack Network
+=============
+
+* 5,250 :doc:`members </slack>` (up from 3,000 members in 2017)
+
+Newsletter
+==========
+
+* 4,707 :doc:`subscribers </newsletter>` (up from 3,406 in 2017)
+
+Meetups
+=======
+
+* 8642 Members (up from 5589 in 2016)
+* 39 :doc:`Meetups </meetups/index>` (up from 30 Meetups in 2017)
+* 4 Continents (same as in 2016)
+
+Podcast
+=======
+
+* 18 episodes (up from 12 episodes in 2017)
+
+Website
+=======
+
+* 450,000 page views (same as 2017)
+* 250,000 sessions (up from 225,000 in 2017)
+* :doc:`Guide </guide/index>`: 225,000 views
+* :doc:`Conference sites </conf/index>`: 120,000 views
+
+This is one place where we haven't seen a lot of growth.
+We haven't added much new content to the guide this past year,
+so I think that is the primary reason that traffic hasn't increased.
```

Comment:
Are the stats for the talk video pages at all meaningful or interesting? 

I'v been wondering about timeboxing work on the docs  Guide, either myself, or all of us or with volunteers. An hour a week or something. The challenge is the org work that needs doing *before* that can meaningfully happen

Reply:
Yea, we don't get a ton of traffic there. The main source of traffic is Google searches for content in the guide. We should do a better job of onboarding those folks into the community tho.